indico/htdocs/js/utils/forms.js

```diff
@@ -77,9 +102,33 @@
             });
         }).on('change input', function() {
             var $this = $(this);
-            var untouched = $this.serialize() == $this.data('initialData');
+            var untouched = $this.serialize() === $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
             $this.find('[data-disabled-until-change]').prop('disabled', untouched);
             $this.closest('form').data('fieldsChanged', !untouched);
+            if ($this.find('[data-save-reminder]').length) {
+                if (!isElementInView($this.find('[data-save-reminder]'))
+                    && !untouched && !$cornerMessage.length) {
+                    showSaveCornerMessage($this);
+                } else if (untouched) {
+                    hideSaveCornerMessage($cornerMessage);
+                }
+            }
+        });
+
+        // Remove corner message when 'Save' button is visible on screen
+        $(window).scroll(function() {
```

Comment:
Running code on scroll may trigger quite often. Maybe use `_.debounce()` to only trigger it once the user stopped scrolling?